README.md

```diff
@@ -324,8 +324,8 @@ interface Observable {
   Observable take(unsigned long long);
   Observable drop(unsigned long long);
   Observable flatMap(Function project);
-  Observable toArray();
-  Observable forEach(Function callback);
+  Promise<sequence<any>> toArray();
+  undefined forEach(Function callback);
```

Comment:
Issue: This should return a `Promise<void>` in TS terms that resolves when the observable completes, or errors if the observable errors.

```suggestion
  Promise<undefined> forEach(Function callback);
```